testController.js

```diff
@@ -1,8 +1,37 @@
 document.addEventListener('DOMContentLoaded', function() {
-    document.getElementById('home').addEventListener('click', function() {
-        console.log('vleze')
-        getWeather('sofia').then(function(data) {
-            console.log(data)
+        function getCity() {
+            return document.getElementById('city').value
+        }
+    document.getElementById('enterCity').addEventListener('click', function() {
+        let city = document.getElementById('city').value 
+        getWeather(getCity()).then(function(data) {
+            let weatherToday = data;
+            document.getElementById('infoWraper').style.display = 'block';
+            document.getElementById('cityName').textContent = `Град: ${weatherToday.name}`;
+            document.getElementById('weatherDescription').textContent = `Днес времето ще бъде ${weatherToday.weather[0].description}`;
+            document.getElementById('weatherTemps').textContent = `Температурата в момента е ${Math.round(weatherToday.main.temp - 273.15)} градуса`
         })
+        
     })
+    document.getElementById('seeForecast').addEventListener('click',function() {
+        
+        getForecast(getCity()).then(function(data) {
+            document.getElementById('forecast').style.display = 'block';
+            document.getElementById('imgDayOne').src = `http://openweathermap.org/img/w/${data.list[0].weather[0].icon}.png`
+            document.getElementById('dayOne').textContent = `Утре времето ще бъде ${data.list[0].weather[0].description}`;
+            document.getElementById('dayOneTemp').textContent = `Със средна температура от ${Math.round(data.list[0].main.temp - 273.15)} градуса`;
```

Comment:
Avoid `magic numbers` in the code (273.15). Consider extracting it in a constant.